py2-pybind11-toolfile.spec

```diff
@@ -1,5 +1,5 @@
 ### RPM external py2-pybind11-toolfile 1.0
```

Comment:
@davidlange6 , rename the spec to `py3-pybind11.spec`  and also update the name in the line above